packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -404,3 +418,17 @@ svg.spectrum-Table-sortedIcon {
 .spectrum-Table-checkbox {
   vertical-align: super;
 }
+
+.spectrum-Table-colResizeIndicator {
```

Comment:
Haven't looked into why this is and its pretty minor, but the resizer is a bit thicker in the sortable columns [story](https://reactspectrum.blob.core.windows.net/reactspectrum/0957aabaea66012ab93157805caf52e75f0fc312/storybook/index.html?path=/story/tableview--allowsresizing-uncontrolled-sortable-columns&providerSwitcher-toastPosition=bottom): 
<img width="289" alt="image" src="https://user-images.githubusercontent.com/9661127/188207491-1d2ec879-e018-4a2d-ad17-86a0f6f54dff.png">

Reply:
thanks for catching that, i'll take a look